src/fidesops/api/v1/exception_handlers.py

```diff
@@ -0,0 +1,20 @@
+from typing import Callable, List
+
+from fastapi import Request
+from fastapi.responses import JSONResponse, Response
+from starlette.status import HTTP_500_INTERNAL_SERVER_ERROR
+
+from fidesops.common_exceptions import FunctionalityNotConfigured
+
+
+class ExceptionHandlers:
+    def functionality_not_configured_handler(
+        request: Request, exc: FunctionalityNotConfigured
+    ) -> JSONResponse:
+        return JSONResponse(
+            status_code=HTTP_500_INTERNAL_SERVER_ERROR, content={"message": str(exc)}
+        )
+
+    @classmethod
+    def get_handlers(cls) -> List[Callable[[Request, Exception], Response]]:
```

Comment:
wanted to provide a starting point if more exception handlers would be needed over time...perhaps there's a more elegant/dynamic way of doing this, having to list out the methods seems a bit cumbersome